lib/matplotlib/tests/test_legend.py

```diff
@@ -1,14 +1,13 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
-import six
 from six.moves import xrange
```

Comment:
If you want to get rid of this too, replace the `list(xrange(#))` by `np.arange(#)`; I don't think there's anything here that explicitly requires a list.